Intermittent clippy failure on unrelated changes

https://github.com/whamcloud/integrated-manager-for-lustre/pull/1947/checks?check_run_id=748961651

```
     Checking iml-gui v0.2.0 (/home/runner/work/integrated-manager-for-lustre/integrated-manager-for-lustre/iml-gui/crate)
error: redundant closure found
##[error]   --> src/components/action_dropdown/mod.rs:249:72
    |
249 |                 let xs = x.entry(r.label().to_string()).or_insert_with(|| vec![]);
    |                                                                        ^^^^^^^^^
    |
    = note: `-D clippy::redundant-closure` implied by `-D warnings`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#redundant_closure

error: redundant closure found
##[error]   --> src/page/servers.rs:207:98
    |
207 | ...                   div![lnet_by_server_view(x, cache, all_locks).unwrap_or_else(|| vec![])]
    |                                                                                    ^^^^^^^^^
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#redundant_closure

error: aborting due to 2 previous errors
##[error]aborting due to 2 previous errors
error: could not compile `iml-gui`.
##[error]could not compile `iml-gui`.
To learn more, run the command again with --verbose.
##[error]The process '/usr/share/rust/.cargo/bin/cargo' failed with exit code 101
```

--- a/iml-gui/crate/src/components/action_dropdown/mod.rs
+++ b/iml-gui/crate/src/components/action_dropdown/mod.rs
@@ -246,7 +246,7 @@ fn group_actions_by_label(xs: Vec<Arc<AvailableAction>>, cache: &ArcCache) -> Ac
     let mut x = xs.into_iter().fold(BTreeMap::new(), |mut x, action| {
         match cache.get_erased_record(&action.composite_id) {
             Some(r) => {
-                let xs = x.entry(r.label().to_string()).or_insert_with(|| vec![]);
+                let xs = x.entry(r.label().to_string()).or_insert_with(Vec::new);
 
                 xs.push((action, r));
             }

--- a/iml-gui/crate/src/page/servers.rs
+++ b/iml-gui/crate/src/page/servers.rs
@@ -203,10 +203,8 @@ pub fn view(
                                 .merge_attrs(class![C.text_center]),
                                 table::td_view(date_view(sd, &x.boot_time)).merge_attrs(class![C.text_center]),
                                 table::td_view(span![x.server_profile.ui_name]).merge_attrs(class![C.text_center]),
-                                table::td_view(
-                                    div![lnet_by_server_view(x, cache, all_locks).unwrap_or_else(|| vec![])]
-                                )
-                                .merge_attrs(class![C.text_center]),
+                                table::td_view(div![lnet_by_server_view(x, cache, all_locks).unwrap_or_else(Vec::new)])
+                                    .merge_attrs(class![C.text_center]),
                                 td![
                                     class![C.p_3, C.text_center],
                                     action_dropdown::view(x.id, &row.dropdown, all_locks, session)

